modules/serverless-testing-nodejs-functions.adoc

```diff
@@ -6,11 +6,18 @@
 [id="serverless-testing-nodejs-functions_{context}"]
 = Testing Node.js functions
 
-Node.js functions can be tested locally on your computer. In the default project that is created when you create a function using `kn func create`, there is a *test* folder that contains some simple unit and integration tests.
+Node.js functions can be tested locally on your computer. In the default project that is created when you create a function by using `kn func create`, there is a *test* folder that contains some simple unit and integration tests.
```

Comment:
I know this is existing text. This sentence seems awkward, but I don't have a much better way of saying it. 
```
You can test your Node.js functions by using the simple unit and integration tests that are created by the `kn func create` command. The tests are in the *test* folder within the default project that is created by the command.
```